cljfmt/src/cljfmt/main.clj

```diff
@@ -191,11 +191,14 @@
 (defn- command-name []
   (or (System/getProperty "sun.java.command") "cljfmt"))
 
+(defn- file-exists? [path]
+  (.exists (io/as-file path)))
+
 (defn -main [& args]
   (let [parsed-opts   (cli/parse-opts args cli-options)
         [cmd & paths] (:arguments parsed-opts)
         options       (merge-default-options (:options parsed-opts))
-        paths         (or (seq paths) default-paths)]
+        paths         (filter file-exists? (or (seq paths) default-paths))]
```

Comment:
This should be:

```clojure
(or (seq paths) (filter file-exists? default-paths))
```

If people add a path manually that doesn't exist, we *do* want to throw an error.